src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -613,6 +613,34 @@ public Locomotive closeWindow() {
         return closeWindow(null);
     }
 
+    /**
+     * Scroll to a specified element
+     *
+     * @param css Css locator for the element to scroll to
+     * @return This instance for method chaining.
+     */
+    public Locomotive scrollTo(String css) {
+        scrollTo(By.cssSelector(css));
+
+        return this;
+    }
+
+    /**
+     * Scroll to a specified element
+     *
+     * @param by Locator for the element to scroll to
+     * @return This instance for method chaining.
+     */
+    public Locomotive scrollTo(By by) {
+        // Find the element to scroll to.
+        final WebElement element = waitForElement(by);
+
+        // Execute javascript to scroll to the element.
+        ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView()", element);
```

Comment:
What's the difference between using this to Selenium `Actions`? I think there is already a method in the `Conductor` class that does this, it's called `moveToElement`, but it uses `Actions`. The class, `waitForElement` uses this move method, which in turn most conductor classes use the `waitForElement` method. 

This is probably a great opportunity to consolidate these methods into one.

Reply:
`Actions.moveToElement()` does not scroll the view of the browser, see https://github.com/w3c/webdriver/issues/1005. The javadoc on the method is currently not correct :(

I agree with the sentiment of calling the new `scrollTo()` inside 'waitForElement`. This would make calls to `waitForElement` much more robust.
I believe calling `moveToElement()` to position the mouse on the middle of the element during every call of `waitForElement()` is not necessary though. For example: to get the text of an element it shouldn't be necessary to change the mouse position. I think it should only be necessary for `click()` calls. Am I missing some details?

Reply:
Yeah that's correct, that's the initial reason I added the `moveToElement` into the `waitForElement` method, selenium would have trouble clicking certain elements if the mouse was not moved first, that and getting better screenshots in general. 

Maybe the `moveToElement` method can be moved to only be used in the `click` method and inside the `waitForElement` replace the `move...` method with the your `scrollTo` method.

Also I'd like to see some unit tests around this :)

Reply:
Sounds good, I will make the necessary changes.